packages/ses/test/tame-date.test.js

```diff
@@ -16,7 +16,9 @@ test('lockdown start Date is powerful', t => {
 });
 
 test('lockdown Date.prototype.constructor is powerless', t => {
-  const SharedDate = Date.prototype.constructor;
+  const SharedDate = /** @type {DateConstructor} */ (
```

Comment:
Is `Date` not `Date.prototype.constructor`? cc @erights

Reply:
No, `SharedDate` is attenuated to avoid a timing side-channel by default, as is `Date.now()` and similarly `Math.random()` for a deterministic PRNG snooping side-channel. The signature is the same.